app/components/trainee_status_card/view.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module TraineeStatusCard
+  class View < GovukComponent::Base
+    attr_reader :state, :user, :target
+
+    STATUS_COLOURS = {
+      draft: "grey",
+      submitted_for_trn: "turquoise",
+      trn_received: "blue",
+      recommended_for_qts: "purple",
+      qts_awarded: "",
+      deferred: "yellow",
+      withdrawn: "red",
+    }.freeze
+
+    def initialize(state:, user:, target:)
+      @state = state
+      @user = user
+      @target = target
+    end
+
+    def count
+      if user.system_admin?
+        Trainee.where(state: state).count
+      else
+        Trainee.where(state: state, provider: user.provider).count
+      end
+    end
+
+    def state_name
+      {
```

Comment:
I think we already have translations for these in locales. You could use these here for maintainability. Something like:

```
def state_name
  I18n.t("activerecord.attributes.trainee.states.#{state}"
end
```

Reply:
Thanks, fixed